genai-perf/README.md

```diff
@@ -365,31 +363,25 @@ a request in order. Random means that assignment is uniformly random
 
 ##### `--backend {tensorrtllm,vllm}`
 
-When using the "triton" service-kind, this is the backend of the model. For the
+When benchmarking Triton, this is the backend of the model. For the
 TRT-LLM backend, you currently must set `exclude_input_in_output` to true in the
 model config to not echo the input tokens in the output. (default: tensorrtllm)
 
 ##### `--endpoint <str>`
 
 Set a custom endpoint that differs from the OpenAI defaults. (default: `None`)
 
-##### `--endpoint-type {chat,completions,embeddings,rankings}`
+##### `--endpoint-type <str>`
```

Comment:
Is it true that the endpoint type can be anything now? I thought there was still a restricted list of choices (based on the mapping)?

Reply:
There is, and the help menu should be able to provide that via the CLI (and I imagine the config does as well). But we have not been keeping this endpoint-type list up-to-date as we keep adding endpoints, so nvclip, multimodal, etc. are not on this list.

I would rather just mark that it is a string. This provides less information, but I would rather not mislead the users, risk our documentation continually getting out-of-step with implementation, or have a very long list here.

Reply:
As long as there is a clear error message if an invalid value is provided and the documentation has the proper values, this should be fine.